mongo/namespace.go

```diff
@@ -100,10 +101,10 @@ func RandomizeNamespaces() (newNamespaces map[string][]string, restore func()) {
 	defer namespacesMu.Unlock()
 	oldNamespaces := map[randomizedName][]randomizedName{}
 	for db, colls := range namespaces {
-		newDBName := randomizedName{ptr: db, from: *db, to: "test-" + utils.RandomAlphaString(5)}
+		newDBName := randomizedName{ptr: db, from: *db, to: "test-" + strings.ToLower(utils.RandomAlphaString(20))}
```

Comment:
To help prevent an engineer from getting confused and removing the "ToLower" requirement, I'd add a quick comment:
```
APP-5672: We choose 20 random lower case characters as a simple heuristic to avoid collisions. We use lower case letters specifically because MongoDB, for historical reasons, disallows creating a database with the same letters but with different capitalization. 
```